engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -950,15 +947,6 @@ public void testCaseWithComment() throws Exception {
     """);
   }
 
-  @Test
-  @Ignore // Crashes old parser
-  public void testAlternationTypes() throws Exception {
-    parseTest("""
-    foo : [Integer | Text] -> (Integer | Text)
-    foo v = v.at 0
-    """);
-  }
-
```

Comment:
We don't need this test (we can't compare it against the old parser); it was just an interesting case

Reply:
So the new one won't crash? Maybe still worth leaving then